jdi-bdd-tests/src/main/java/io/github/com/sections/ContactForm.java

```diff
@@ -8,14 +8,15 @@
 import com.epam.jdi.light.ui.html.elements.common.Checkbox;
 import com.epam.jdi.light.ui.html.elements.common.TextArea;
 import com.epam.jdi.light.ui.html.elements.common.TextField;
+import com.epam.jdi.light.ui.html.elements.complex.DataListOptions;
 import io.github.com.entities.Contacts;
 
 
 public class ContactForm extends Form<Contacts> {
 	TextField name, lastName, position, passportNumber, passportSeria;
 
 	Dropdown gender;
-	IsCombobox religion;
+	DataListOptions religion;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)